lib/Conversion/TritonGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -57,7 +57,7 @@ struct DotOpMFMAConversionHelper {
           loc, TypeRange{resType},
           ValueRange{valA, valB, valC, zeroFlag, zeroFlag, zeroFlag});
     case MatrixCoreType::FP32_BF16_BF16_FP32:
-      return rewriter.create<ROCDL::mfma_f32_32x32x4bf16>(
+      return rewriter.create<ROCDL::mfma_f32_32x32x8bf16_1k>(
```

Comment:
Since this only works on MI200, do we need to guard to distinguish MI100 and MI200?

Reply:
This switch is implemented in #295 (the thing with kWidth)
I tried to make #294 PR as simple as possible, so we can merge it asap, because I was told that MI200 support is in priority.

Reply:
Got it, thanks.